frontend/src/components/maps/MinePin.js

```diff
@@ -134,6 +134,7 @@ export class MinePin extends Component {
 
         const fcl = FlareClusterLayer.FlareClusterLayer(options);
         fcl.title = "Mine Pins";
+        fcl.refreshOnStationary = true;
```

Comment:
This seems to be true by default. Did you notice some funky behavior here?

Reply:
you are right I was just playing around with that and forgot to take it out